lib/rubocop/cop/root_cops/dont_mix_keyword_and_positional_args.rb

```diff
@@ -0,0 +1,19 @@
+module RuboCop
+  module Cop
+    module RootCops
+      class DontMixKeywordAndPositionalArgs < Cop
```

Comment:
The style guide for cops is to include a comment with an example of good/bad usage. For example

```ruby
      # prefer `Enumerable#detect` over `Enumerable#find`
      #
      # @example
      #   # bad
      #   [1,2,3].find { |x| x.even? }
      #   [1,2,3].find(&:even?)
      #
      #   # good
      #   [1,2,3].detect { |x| x.even? }
      #   [1,2,3].detect(&:even?)

      class UseDetect < Cop
# ...
```